airbyte-server/src/test/java/io/airbyte/server/services/AirbyteGithubStoreTest.java

```diff
@@ -45,15 +46,55 @@
   private static MockWebServer webServer;
   private static AirbyteGithubStore githubStore;
 
-  @BeforeAll
-  public static void setUp() {
+  @BeforeEach
```

Comment:
this is now BeforeEach since we turn off the server during some tests to mimic bad internet